IQSS/dataverse-frontend
# 263 - Add file metadata section to the File page

## What this PR does / why we need it:
This PR adds the file metadata section UI of the File Page.

## Which issue(s) this PR closes:

- Closes #263 

## Special notes for your reviewer:

## Suggestions on how to test this:
1. Install the necessary dependencies using `npm install`.
5. Build the design system with `cd packages/design-system && npm run build`.
6. Navigate back to the root directory with `cd ../../`.
7. Start Storybook with `npm run storybook`.
8. Visit [the File Page in the Storybook](http://localhost:6006/?path=/story/pages-file--default) and check that the File Metadata section looks as expected

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
File Metadata section added to the File Page
![Captura de pantalla 2023-12-14 a las 11 38 09](https://github.com/IQSS/dataverse-frontend/assets/23359572/ece0633e-c87a-42e2-a06a-0f77dd082b7d)

## Is there a release notes update needed for this change?:
File Metadata section added to the File Page
## Additional documentation:
